pkg/project/auth/cache.go

```diff
@@ -67,6 +67,35 @@ func subjectRecordKeyFn(obj interface{}) (string, error) {
 	return subjectRecord.subject, nil
 }
 
+type State interface {
```

Comment:
Correct, this is a means of knowing if any reflector being utilized has changed since the last interval.

Will think of better name, concede your point.